api/src/utils/error-formatting.ts

```diff
@@ -47,6 +50,30 @@ export const formatProjectCompletedValidation = (
       };
 };
 
+/**
+ * Format validation errors for /project-completed.
+ *
+ * @param errors An array of validation errors.
+ * @returns Formatted errors that can be used in the response.
+ */
+export const formatCertificationValidation = (
+  errors: ErrorObject[]
+): FormattedError => {
+  const error = getError(errors);
+
+  return error.instancePath === '' &&
+    Object.hasOwn(certTypes, error.params.missingProperty as CertLogs)
```

Comment:
hasOwn checks the `properties`, I need to use [valus](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values)